src/slice/mod.rs

```diff
@@ -85,10 +85,14 @@ pub trait ParallelSliceMut<T: Send> {
         }
     }
 
-    /// Sorts the slice.
+    /// Sorts the slice in parallel.
     ///
     /// This sort is stable (i.e. does not reorder equal elements) and `O(n log n)` worst-case.
     ///
+    /// When applicable, unstable sorting is preferred because it is generally faster than stable
+    /// sorting and it doesn't allocate auxilliary memory.
```

Comment:
I think GitHub is acting up... I pushed a new commit but it's not yet visible. Anyways, if you don't see the 6th commit in this PR, you can find it in the [original branch](https://github.com/stjepang/rayon/tree/par-sort). :/